app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/image_import_workflow.rb

```diff
@@ -42,13 +42,19 @@ def post_execute
 
     ems = ExtManagementSystem.find(options[:ems_id])
 
+    cos_pvs_creds = options[:cos_pvs_creds]
+    bearer = Authentication.find(cos_pvs_creds[:bearer_id])
+
     body = {
       :source        => 'url',
       :imageName     => options[:miq_img][:name],
       :osType        => options[:miq_img][:os],
       :diskType      => options[:diskType],
       :imageFilename => "#{options[:session_id]}.ova",
-      **options[:cos_pvs_creds]
+      :region        => cos_pvs_creds[:region],
+      :bucketName    => cos_pvs_creds[:bucketName],
+      :accessKey     => bearer[:userid],
+      :secretKey     => ManageIQ::Password.try_decrypt(bearer[:password])
```

Comment:
```suggestion
      :accessKey     => bearer.userid.,
      :secretKey     => ManageIQ::Password.try_decrypt(bearer.password)
```

Reply:
Actually I'm confused by the try_decrypt.  The password should be auto-decrypted here.  Perhaps because you weren't going through the accessor methods?

Reply:
It isn't decrypted at that stage, I've checked it.